### PR TITLE
[#164046204] configure TravisCI

### PR DESCRIPTION
**Pull request by MandelaK**
**Description**

This **PR** configures TravisCI for Continuous Integration
<hr>
**Type of change**

- [x] chore(non-breaking change).

<hr>

**Checklist:**
- [x] This follows all the guidelines for this project.

#### PT Stories

[#164046204](https://www.pivotaltracker.com/story/show/164046204)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,10 +27,10 @@
     "not op_mini all"
   ],
   "devDependencies": {
-    "eslint": "^5.16.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.16.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-react": "^7.12.4"
+    "eslint-plugin-react": "^7.12.4",
+    "fsevents": "^1.2.7"
```
Comment (abulojoshua1) [inline package.json:34]: @MandelaK, what does this dependency do?

**Review by cob04 — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (MandelaK) [reply package.json:34]: Tests aren't running on some machines without it ... There are two ways to fix it, either install watchman globally using brew and also install xcode, or you can install fsevents as a dev dependency ... I read about this fix here https://github.com/expo/expo/issues/854#issuecomment-343243804 and informed the team about the fix in both @BrianSerem's PR and in the slack channel

**Review by RachelleMaina — APPROVED** [review]
LGTM

**Review by ElMonstro — APPROVED** [review]

**Review by abulojoshua1 — APPROVED** [review]
LGTM

**Review by abulojoshua1 — APPROVED** [review]

**Pull request merged**